### PR TITLE
Added info on `UMB_SESSION` cookie

### PR DESCRIPTION
**Pull request by eshanrnh**
Added info as per https://umbraco.slack.com/archives/C02JJQU9B/p1655981011967159?thread_ts=1655971280.012539&cid=C02JJQU9B

### CODE REVIEW

File: Reference/Security/Cookies/index.md [hunk 1]
```diff
@@ -20,5 +20,17 @@ The below cookies are necessary while accessing the Umbraco Backoffice for funct
...
+```cs
+services.AddSession(options =>
+    {
+        options.Cookie.Name = "UMB_SESSION";
+        options.Cookie.HttpOnly = true;
+        options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
+    });
+```
+
+For information on the rest of the cookies, see the [Constants-Web.cs](https://github.com/umbraco/Umbraco-CMS/blob/5bfab13dc5a268714aad2426a2b68ab5561a6407/src/Umbraco.Core/Constants-Web.cs) file on Github.
```
Comment (sofietoft) [inline Reference/Security/Cookies/index.md:36]: The link references a specific branch on the CMS repo.
Shouldn't it link to the default branch instead?

This is the current default branch: https://github.com/umbraco/Umbraco-CMS/blob/v10/contrib/src/Umbraco.Core/Constants-Web.cs

**Review by sofietoft — COMMENTED** [review]
Looks great 💪 

I've added a single note about the link to the CMS repo.

File: Reference/Security/Cookies/index.md (hunk 1 above)
Comment (eshanrnh) [reply Reference/Security/Cookies/index.md:36]: Good point 💪 I'll change it right away.

**Comment by eshanrnh** [comment]
Merging it in.

**Pull request merged**